### PR TITLE
Fix UI :  Google login layout style

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes :

-  Google Login style

#
### Type of change :

- [x] Bug fix

#
### Frontend Preview (Screenshots) :
Screenshot 2022-09-24 at 2 34 05 PM

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/styles/x-master.css [hunk 1]
```diff
@@ -257,6 +257,20 @@ pre {
   background-color: #e8e7e4;
 }
 
+/* google sso layout style  */
+.googlecontainer {
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/styles/x-master.css:261]: Can we move this class to `app.less` file?

Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/styles/x-master.css]: @Sachin-chaurasiya  updated

**Review by Sachin-chaurasiya — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/src/styles/app.less [hunk 2]
```diff
@@ -197,3 +197,17 @@
...
+  height: 580px;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  flex-direction: column;
+
+  span {
+    margin: 0;
+  }
+}
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/styles/app.less:213]: App.less is for utils classes and not for specific use case classes I would recommend creating `sso.less` or a similar file and use it, but not this!

**Review by ShaileshParmar11 — COMMENTED** [review]
Left one comment

**Pull request merged**